net/socket/udp_socket_starboard.cc

```diff
@@ -132,7 +132,8 @@ void UDPSocketStarboard::Close() {
   write_callback_.Reset();
   send_to_address_.reset();
 
-  bool ok = socket_watcher_.StopWatchingSocket();
+  bool ok = socket_watcher_.UnregisterInterest(kSbSocketWaiterInterestRead |
```

Comment:
same here, why not use StopWatchingSocket?

Reply:
Done.